### PR TITLE
PL-953: Updated documentation for several endpoints.

### PR DESCRIPTION
**Pull request by mujeeb-bboxx**

### CODE REVIEW

File: source/includes/_appliance_part_type_replacement_linker.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Field | Description
 __appliance_part_type_replacement_linker_id__ <br><font color="DarkGray">_int_</font> <font color="Crimson">__(primary key)__</font> | A unique integer identifier for each appliance_part_type_replacement_linker.
 __<a href="/#appliance-type">appliance_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#existing-appliance-part-type">existing_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
+__max_part_count__  <br><font color="DarkGray">_int_</font> | count of maximum parts that can be changed
```
Comment (barkatkhan-bboxx) [inline source/includes/_appliance_part_type_replacement_linker.md]: Please add this description: Maximum number of parts of this type that can be replaced for this appliance type

Comment (barkatkhan-bboxx) [reply source/includes/_appliance_part_type_replacement_linker.md]: Or copy this line of code
`__max_part_count__  <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,default=1)</font>| Maximum number of parts of this type that can be replaced for this appliance type`

File: source/includes/_appliance_repair_appliance_part_type_linker.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ __appliance_repair_appliance_part_type_linker_id__ <br><font color="DarkGray">_i
 __<a href="/#appliance-repair">appliance_repair_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#old-appliance-part-type">old_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#new-appliance-part-type">new_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(foreign-key)</font> | 
+__repaired_parts_count__ <br><font color="DarkGray">_int_</font> <font color="Crimson"></font> | count of repaired parts
```
Comment (barkatkhan-bboxx) [inline source/includes/_appliance_repair_appliance_part_type_linker.md]: Please change to this:
`__repaired_parts_count__  <br><font color="DarkGray">_int_</font> | Number of parts of this type replaced in this repair`

File: source/includes/_products.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ __latest_rtc_connection_history_id__ <br><font color="DarkGray">_int_</font> <fo
 __latest_rtc_mo_status_report_time__ <br><font color="DarkGray">_datetime_</font> <font color="Crimson"></font> | 
 __latest_rtc_mt_status_update_time__ <br><font color="DarkGray">_datetime_</font> <font color="Crimson"></font> | 
 __rtc_enabled_flag__ <br><font color="DarkGray">_boolean_</font> <font color="Crimson">(not-null)</font> | 
+__dcm_enabled_flag__ <br><font color="DarkGray">_boolean_</font> <font color="Crimson">(not-null)</font> | default value is True for all BP20 devices
```
Comment (barkatkhan-bboxx) [inline source/includes/_products.md]: Please define in the description, what dcm_enabled_flag is.

> For say, It shows the unit is DCM enabled

File: source/includes/_appliance_part_type_replacement_linker.md (hunk 1 above)
Comment (mujeeb-bboxx) [reply source/includes/_appliance_part_type_replacement_linker.md]: `max_part_count` is nullable field. Please see the response below,
```
{
    "appliance_part_type_replacement_linker_id": 521,
    "appliance_type_id": 27,
    "created_at": "2020-07-20T16:55:23.257503",
    "created_by": "w.theaker@bboxx.co.uk",
    "existing_appliance_part_type_id": 510,
    "max_part_count": null,
    "modified_at": null,
    "modified_by": null,
    "replacement_appliance_part_type_id": 510
}
```

Updating the line to this instead,
`__max_part_count__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(default=1)</font>| Maximum number of parts of this type that can be replaced for this appliance type`

**Review by barkatkhan-bboxx — APPROVED** [review]

**Review by z-adeel-bboxx — APPROVED** [review]

**Pull request merged**